server/src/main/java/io/spine/server/entity/storage/AsEntityRecordColumn.java

```diff
@@ -69,10 +71,32 @@ private AsEntityRecordColumn() {
      */
     @SuppressWarnings("BadImport")       // `create` looks fine in this context.
     static <V> RecordColumn<EntityRecord, V> apply(Column<?, ?> original, Class<V> typeOfValues) {
+        return apply(original, typeOfValues, new NoGetter<>());
+    }
+
+    /**
+     * Creates a view on the given column as on a record column of an {@link EntityRecord}
+     * with the given type of values.
+     *
+     * <p>A value of the resulting column is determined by the passed getter.
+     *
+     * @param original
+     *         the column to create a view for
+     * @param typeOfValues
+     *         the type of the column values
+     * @param getter
+     *         the getter returning the value of the resulting column view
+     * @param <V>
+     *         the type of the column values
+     * @return a view on the column
+     */
+    @SuppressWarnings("BadImport")       // `create` looks fine in this context.
```

Comment:
Assuming it's used only once, I think it would be less trouble not to use the bad import. Now it requires a reader to link the warning suppression, with the static import statement, and then with the usage in the `return` statement (also pipping at the method signature about the returned type). Just using `RecordColumn.create()` would be easier to understand, and one line less of code. Please consider.